src/textual/message_pump.py

```diff
@@ -75,6 +75,19 @@ def __new__(
                         f"on_{namespace}_{camel_to_snake(value.__name__)}"
                     )
 
+        # Look for reactives with public AND private compute methods.
+        for attr_name, value in class_dict.items():
+            if attr_name.startswith("compute_") and callable(value):
+                reactive_name = attr_name[len("compute_") :]
```

Comment:
Possibly taking things too far, but I think I'd be tempted to take the `len` of the string outside of the loop and use that, so you're not calling `len` every time around.

Reply:
So this is what it feels like to be Rodrigoed!

Reply:
I learnt from the best. 🤷‍♀️